x-pack/plugins/security/server/plugin.ts

```diff
@@ -186,6 +186,20 @@ export class SecurityPlugin
   };
 
   constructor(private readonly initializerContext: PluginInitializerContext) {
+    this.authenticationService = new AuthenticationService(
+      this.initializerContext.logger.get('authentication')
+    );
+    this.auditService = new AuditService(this.initializerContext.logger.get('audit'));
+    this.elasticsearchService = new ElasticsearchService(
+      this.initializerContext.logger.get('elasticsearch')
+    );
+    this.sessionManagementService = new SessionManagementService(
+      this.initializerContext.logger.get('session')
+    );
+    this.anonymousAccessService = new AnonymousAccessService(
+      this.initializerContext.logger.get('anonymous-access'),
+      this.getConfig
+    );
```

Comment:
This seems to be a result of a merge conflict that was resolved incorrectly?

Reply:
Good catch, removed in 65adb76b64f34fb43894ddd53898a41a54877898